ghcide/test/data/multi-unit/cabal.project

```diff
@@ -0,0 +1,2 @@
+packages: a b c
+multi-repl: True
```

Comment:
Wat?

Reply:
Probably the new cabal feature that uses multiple home-units when loading them into a repl?

Reply:
Yes, @fendor is right, I generated the `-unit` files by using a special branch of cabal with the new feature and then editing it a bit.

This cabal.project isn't necessary or even useful in this patch so I will remove it.

Reply:
I am interested to know *what* branch and what @mpickering is planning there, and whether we still need the work from https://github.com/haskell/cabal/pull/7500, or whether the changes done in that fork render `cabal status` command irrelevant?
Maybe we can briefly discuss that at the next meeting.

Reply:
Here is the branch, it is still a WIP: https://github.com/mpickering/cabal/tree/wip/no-configure